build/critical.go

```diff
@@ -13,4 +14,5 @@ func Critical(v ...interface{}) {
 		panic(s)
 	}
 	os.Stderr.WriteString(s)
+	debug.PrintStack()
```

Comment:
did you know that `println` prints to stderr?
stderr is the proper place for error output. The streams are merged on the terminal. In some cases, stderr might be _more_ likely to be see, e.g. when piping stdout to a file, the stderr output will not be piped and instead displayed in the terminal.